src/rt-app.c

```diff
@@ -1174,7 +1180,7 @@ void *thread_body(void *arg)
 	/* Force thread into root taskgroup. */
 	reset_thread_taskgroup();
 
-	if (timings) {
+	if (opts.logsize && timings) {
```

Comment:
Why have you added the test on opts.logsize ?
timings is null if logsize is <= 0